qa/comparitor.py

```diff
@@ -0,0 +1,180 @@
+import csv
```

Comment:
Can we move this `qa` package to `libs/qa`?  I think it's confusing to have a lot of top level packages (i already don't love that pyseir/ + api/ is outside of libs) and not clear why some code is top level vs others not.